packages/fluentui/react-northstar-emotion-renderer/src/invokeKeyframes.ts

```diff
@@ -32,7 +32,7 @@ const keyframes = (
  */
 export function invokeKeyframes(cache: EmotionCache, styles: ICSSInJSStyle) {
   for (const property in styles) {
-    if (isStyleObject(styles[property])) {
+    if (isStyleObject(styles[property as keyof ICSSInJSStyle])) {
```

Comment:
to avoid the aggressive casting you could just define `property` as a variable before the look typed to `keyof ICSSInJSStyle`

Reply:
Good call, applied ✅